### PR TITLE
XSIAM ransomware playbooks

### PR DESCRIPTION
**Pull request by melamedbn**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/46338

## Description
New ransomware response playbooks.

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0
- [x] 6.6.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

**Comment by altmannyarden** [comment]
@richardbluestone another XSIAM playbook :) thank you

**Comment by richardbluestone** [comment]
Docs review is done. I noticed in the playbook-`Ransomware_Enrich_and_Contain.yml` there is a typo in line 839: 
image

File: Packs/Core/.secrets-ignore [hunk 1]
```diff
@@ -77,4 +77,5 @@ https://test.org
 abuse@w.com
 https://us-cert.cisa.gov/tlp
 fake.url.com
-a@a.gmail.com
```
Comment (altmannyarden) [inline Packs/Core/.secrets-ignore:80]: Do you remember why we added this?

File: Packs/Core/Playbooks/playbook-Ransomware_Enrich_and_Contain_README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+This playbook is responsible for alert data enrichment and response.
```
Comment (altmannyarden) [inline Packs/Core/Playbooks/playbook-Ransomware_Enrich_and_Contain_README.md:1]: This playbook is responsible for **ransomware** alert data enrichment and response.
(not any alert :) )

File: Packs/Core/.secrets-ignore (hunk 1 above)
Comment (melamedbn) [reply Packs/Core/.secrets-ignore:80]: Nope

File: Packs/Core/Playbooks/playbook-Ransomware_Response.yml [hunk 3]
```diff
@@ -0,0 +1,1399 @@
...
+    simple: "True"
+  required: false
+  description: Whether to isolate the endpoints identified with similar IoCs to the
+    ransomware alert.
+  playbookInputQuery:
+- key: RunAdvancedAnalysis
+  value:
+    simple: "True"
+  required: false
+  description: Whether to execute the Ransomware Advanced Analysis playbook.
```
Comment (altmannyarden) [inline Packs/Core/Playbooks/playbook-Ransomware_Response.yml:1385]: lets add in the description that the Advanced Analysis playbook contains manual actions

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**